src/styles/sass/main.scss

```diff
@@ -293,12 +293,15 @@ a {
   }
 }
 
+.contents-view {
+  overflow: auto;
+}
```

Comment:
ok. really we want that to get rid of this all together... so TOC has `overflow: scroll` and _settings has `overflow: hidden`